custom_components/waste_collection_schedule/waste_collection_schedule/source/maidstone_gov_uk.py

```diff
@@ -0,0 +1,93 @@
+import json
+import requests
+
+from datetime import datetime
+from time import time_ns
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+# Many thanks to dt215git for their work on the Bexley version of this provider which helped me write this.
+
+TITLE = "Maidstone Borough Council"
+DESCRIPTION = "Source for maidstone.gov.uk services for Maidstone Borough Council."
+URL = "https://maidstone.gov.uk"
+TEST_CASES = {
+    "Test_001": {"uprn": "10022892379"}, # has mutliple collections on same week per bin type
+    "Test_002": {"uprn": "10014307164"}, # has duplicates of the same collection (two bins for this block of flats?)
+    "Test_003": {"uprn": "200003674881"} # has garden waste collection, at time of coding
```

Comment:
Could you please make one of them integer values and test if your code still works. Users should not care weather to use integer or string values.

Reply:
Oops forgot about that, now done and tested

```
Testing source maidstone_gov_uk ...
  found 3 entries for Test_001
    2023-06-06 : Black bin and food [mdi:trash-can]
    2023-06-09 : Black bin and food [mdi:trash-can]
    2023-06-13 : Recycling bin and food [mdi:recycle]
  found 2 entries for Test_002
    2023-06-07 : Black bin and food [mdi:trash-can]
    2023-06-14 : Recycling bin and food [mdi:recycle]
  found 3 entries for Test_003
    2023-06-06 : Black bin and food [mdi:trash-can]
    2023-06-13 : Recycling bin and food [mdi:recycle]
    2023-06-06 : Garden bin [mdi:leaf]